target/linux/ramips/image/mt7621.mk

```diff
@@ -709,6 +709,17 @@ define Device/xiaomi_mir3p
 endef
 TARGET_DEVICES += xiaomi_mir3p
 
+define Device/xiaoyu_xy-c5
+  MTK_SOC := mt7621
+  IMAGE_SIZE := 32448k
+  DEVICE_VENDOR := XiaoYu
+  DEVICE_MODEL := XY-C5
+  DEVICE_PACKAGES := \
+	kmod-ata-core kmod-ata-ahci \
+	kmod-usb3 kmod-usb-ledtrig-usbport
```

Comment:
There's no usb led so kmod-usb-ledtrig-usbport should be dropped here. The rest three packages fit in one line so the line breaking here is also unnecessary.

Reply:
ok I just used work led to indicate usb link for my self, this is unnecessary for everyone use.